server/weaverbird/backends/pipeline_translator/pipeline_translator.py

```diff
@@ -0,0 +1,43 @@
+from typing import Tuple
+
+from weaverbird.pipeline import PipelineWithVariables, Pipeline, PipelineStep
+
+from server.weaverbird.backends.pipeline_translator.types import QueryRetriever, PipelineTranslationReport, \
+    StepTranslationReport
+from server.weaverbird.backends.pipeline_translator.steps import steps_translators
+
+
+def translate_pipeline(
+    pipeline_to_translate: Pipeline,
+    query_retriever: QueryRetriever
+) -> Tuple[str, PipelineTranslationReport]:
+    """ For now, steps_translators are only SQL steps translators """
```

Comment:
I'm nit sure I understand this docstring. Do you mean than not all steps are implemented?

Reply:
I will clarify. I was meaning, for now the only translator we will implement is a SQL one. 
But maybe in the future we'll have other ones :)

Reply:
Ah OK!
But it makes no sense to have a translator that would use a sql implementation for one step and another like mongo for another step. It seems to me that, at least for now, we don't mix translators. So inside the `SQLTransaltor`, I think it's logical that there will always be only SQL steps.